### PR TITLE
Add dottyEnable helper.

### PR DESCRIPTION
**Pull request by olafurpg**
It's handy to have a one-liner to enable/disable dotty. For
example if you need to

1. disable DottyPlugin, get compiler errors
2. run scalafix
3. enable DottyPlugin, no more compiler errors

### CODE REVIEW

File: src/main/scala/com.felixmulder.dotty.plugin/DottyPlugin.scala [hunk 1]
```diff
@@ -4,6 +4,10 @@ import sbt._
 import sbt.Keys._
 
 object DottyPlugin extends AutoPlugin {
+  // http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22ch.epfl.lamp%22%20dotty
+  val dottyVersion: String =
+    sys.env.getOrElse("COMPILERVERSION", "0.1.1-20170214-606e36b-NIGHTLY")
```
Comment (olafurpg) [inline src/main/scala/com.felixmulder.dotty.plugin/DottyPlugin.scala:9]: Should I maybe also update this constant to the latest nightly?

**Review by olafurpg — COMMENTED** [review]
Review @felixmulder

**Comment by felixmulder** [comment]
Sorry @olafurpg - this got lost in the wealth of notifications this week. Is there something here you need or should we wait for RC4, which will be released after merging #4 ?

**Comment by olafurpg** [comment]
@felixmulder I resolved the conflict in build.sbt it would be nice to get the `dottyEnable` helper in before the next release, at the moment I copy-paste it all over the place. @liufengyun has also found this helper is useful.

**Comment by olafurpg** [comment]
Superseded by #4

**Pull request closed without merging**